connectors/sources/servicenow.py

```diff
@@ -229,6 +329,13 @@ def __init__(self, configuration):
         self.concurrent_downloads = self.configuration["concurrent_downloads"]
         self.servicenow_client = ServiceNowClient(configuration=configuration)
 
+        self.valid_services = None
+        self.invalid_services = None
```

Comment:
```suggestion
        self.valid_services = []
        self.invalid_services = []
```

suggestion: I would use empty lists as default so we're not running into any potential issues, when we try to log/format `None` somewhere else.